ext/pdo/pdo_stmt.c

```diff
@@ -2252,36 +2252,56 @@ static union _zend_function *dbstmt_method_get(
 	char *lc_method_name;
 #if PHP_API_VERSION >= 20041225
 	zval *object = *object_pp;
+	zend_function *fbc_fallback = NULL;
 #endif
 
 	lc_method_name = emalloc(method_len + 1);
 	zend_str_tolower_copy(lc_method_name, method_name, method_len);
 
-	if (zend_hash_find(&Z_OBJCE_P(object)->function_table, lc_method_name, 
+	if (zend_hash_find(&Z_OBJCE_P(object)->function_table, lc_method_name,
 			method_len+1, (void**)&fbc) == FAILURE) {
+
+#if PHP_API_VERSION >= 20041225
+		/* Prepare fallback by running standard method for functions.
+		 * If the function does not find a method to instanciate it will return this fallback method. */
+		fbc_fallback = std_object_handlers.get_method(object_pp, method_name, method_len, key TSRMLS_CC);
+#endif
+
 		pdo_stmt_t *stmt = (pdo_stmt_t*)zend_object_store_get_object(object TSRMLS_CC);
 		/* instance not created by PDO object */
 		if (!stmt->dbh) {
+
+#if PHP_API_VERSION >= 20041225
+			fbc = fbc_fallback;
+#endif
+
 			goto out;
 		}
 		/* not a pre-defined method, nor a user-defined method; check
 		 * the driver specific methods */
 		if (!stmt->dbh->cls_methods[PDO_DBH_DRIVER_METHOD_KIND_STMT]) {
-			if (!pdo_hash_methods(stmt->dbh, 
+			if (!pdo_hash_methods(stmt->dbh,
 				PDO_DBH_DRIVER_METHOD_KIND_STMT TSRMLS_CC)
 				|| !stmt->dbh->cls_methods[PDO_DBH_DRIVER_METHOD_KIND_STMT]) {
+
+#if PHP_API_VERSION >= 20041225
```

Comment:
Also, because 5.5 is 20121113 and this is the smallest supported PHP version, these checks are not needed there.